packages/EasyErrorHandler/src/Bridge/Symfony/Builder/ApiPlatformValidationErrorResponseBuilder.php

```diff
@@ -32,11 +34,19 @@ public static function supports(Throwable $throwable): bool
     {
         $message = $throwable->getMessage();
 
+        // TODO: refactor in 5.0. Use the ApiPlatform\Symfony\Bundle\ApiPlatformBundle class only.
+        $invalidArgumentExceptionClass = stdClass::class;
```

Comment:
Can we set it to `null` instead?

Reply:
Yes.

Reply:
Refactored.

Reply:
During updating the API platform on Payments, I have an ambitious plan to make the API platform return a validation error. Then we can normally rethink this and similar classes.

Reply:
We need both, in version `2.7` `InvalidArgumentException` at least uses in  `\ApiPlatform\Serializer\AbstractItemNormalizer::denormalizeCollection`.